pages/blog/2020-03-12-nextjs-without-client-side-react.md

```diff
@@ -0,0 +1,164 @@
+# Server Side Only React
+
+---
+
+<Intro>
+A commonly heard critique of React is that it's an over-engineered abstraction, that for the vast majority of websites provide little value. I tend to agree, so I removed it... on the client.
```

Comment:
that for the vast majority of websites provide little value -> majority of websites it provides little value